### ISSUE TITLE
Run smoke tests on scripts as part of CI

### ISSUE THREAD

**Issue by patcon** [GCTC-NTGC/gc-digital-talent#2536]
🎟️ **Re-ticketed from:** https://github.com/GCTC-NTGC/gc-digital-talent/pull/2525#issuecomment-1104248612
🎯 **Success Criteria:** The everyday scripts we use (in `maintenance/scripts`) should each be able to run successfully in CI smoke test, and a working app should be minimally validated after each script run.

Thanks to @petertgiles's clever find in #2525, we're now able to run our existing docker setup on GitHub Actions. This means:
1. we can start smoke testing all our scripts, to know whether they've broken or fail to spin up a working app, and
2. we can now implement a MVP Cypress end-to-end test workflow right away, since we can deploy the existing docker setup on GitHub Actions.

The simplest version of this is to just:
1. spin up the bare docker containers
2. ping the site and confirm failure
3. run `setup.sh`, then ping each app and confirm minimal success via 200 response
4. run each `refresh_*.sh`, and confirm appropriate app still works

Opportunities in future: (not expected to be in first PR)
- use GitHub container repo to build containers once, and other jobs just pull from there (current build only take 1m20s, so not much savings here)
- use [build matrixes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-running-multiple-versions-of-nodejs) to run the scripts in parallel (but each would spin up its own docker containers, and need to run setup.sh first which is 6m build time, so maybe not worth the savings)

**Comment by petertgiles** [comment]
Hi @patcon .  Since you're pulling in an issue that is not part of the current sprint backlog could you please:
- assign yourself as owner
- add the interruption label
- add the current sprint
- add an estimate

**Comment by patcon** [comment]
Sure! And is there anything process-wise you'd recommend? Specifically if I prioritized this too unilaterally or did things in the wrong order?

(My thought process was that we kicked this to another issue in #2525, and that this gets us more directly to #1762, which I understood to be the reason we're prioritizing #2217 in current backlog. So this made sense in my mind, but I realize my process of checking in with others might need work 🙏 )

**Comment by petertgiles** [comment]
> And is there anything process-wise you'd recommend?

Hehe, I'm not an agile expert but I think the recommendation is for devs to prioritize the issues in the sprint backlog.  :wink:  

But if you feel an issue is urgent enough to pull into the sprint (or we run out like last sprint!) you can use the interruption label so that it can be tracked and you'll need to add all the meta tags yourself.

**Comment by patcon** [comment]
Closed in #2538
